### DISCUSSION TITLE
Unable to update template

### DISCUSSION THREAD

**Discussion by shelu16** [projectdiscovery/nuclei-templates#7227] [Q&A] [answered]
[WRN] failed to update nuclei templates: [:RUNTIME] failed to install templates at /root/nuclei-templates <- [:RUNTIME] gh auth failed try unsetting GITHUB_TOKEN env variable <- GET https://api.github.com/repos/projectdiscovery/nuclei-templates/releases/latest: 401 Bad credentials []

![image](https://user-images.githubusercontent.com/30596083/234337261-6b21de02-fcf8-4c2b-851b-92635710a5c5.png)

**Comment by princechaddha** [comment, marked as answer]
Hi @shelu16, Thank you for taking the time to create this issue and for contributing to this project 🍻

This warning indicates that there was a failure to update nuclei templates due to a runtime issue with installing templates at the specified directory. The error message suggests that the problem may be related to a failed authentication attempt with GitHub. To resolve the issue, try unsetting the GITHUB_TOKEN environment variable and attempting the update again.

```
unset GITHUB_TOKEN
```

**Comment by shelu16** [comment]
ok thx

**Comment by cdn-185-199-109-15** [comment]
command for that

**Reply by MetzinAround** [reply]
the command for unsetting a github token is 

`unset GITHUB_TOKEN`
